client/gutenberg/extensions/related-posts/edit.jsx

```diff
@@ -80,22 +144,37 @@ class RelatedPostsEdit extends Component {
 			},
 		];
 
-		const postsToDisplay = posts.length ? posts : [];
-		const displayPosts = postsToDisplay.slice( 0, postsToShow );
+		// To prevent the block from crashing, we need to limit ourselves to the
+		// posts returned by the backend - so if we want 6 posts, but only 3 are
+		// returned, we need to limit ourselves to those 3 and fill in the rest
+		// with placeholders.
```

Comment:
Hmm, is this going to reflect what users are going to see on the frontend side?

Reply:
If we update the WPCOM backend after calypso, yes.

Reply:
The reality of the matter is I'm just documenting a possible quirk.